hedera-node/configuration/previewnet/upgrade/throttles.json

```diff
@@ -181,11 +181,24 @@
           "opsPerSec": 0,
           "milliOpsPerSec": 1000000000,
           "operations": [
-            "CryptoGetAccountBalance",
             "TransactionGetReceipt"
           ]
         }
       ]
+    },
+    {
+      "burstPeriod": 0,
+      "burstPeriodMs": 1000,
+      "name": "BalanceQueryLimits",
+      "throttleGroups": [
+        {
+          "opsPerSec": 0,
+          "milliOpsPerSec": 1000000,
+          "operations": [
+            "CryptoGetAccountBalance"
+          ]
+        }
+      ]
     }
   ]
 }
```

Comment:
Can you please check that all throttle.json files have an empty line at the end? We had some issues in the past (with other files) and decided to make this mandatory for all files.

Reply:
Done 👍